app/helpers/routes/sessionHelper.js

```diff
@@ -24,6 +24,7 @@ export const sessionKeys = {
   selectEstimationPeriod: 'selectEstimationPeriod',
   selectedQuantity: 'selectedQuantity',
   orderItems: 'orderItems',
+  catalogueItemAlreadyExist: 'catalogueItemAlreadyExist',
```

Comment:
I think either `catalogueItemExists` or `isExistingCatalogueItem` might be slightly better name.

Reply:
changed to catalogueItemExists